api/src/pcapi/core/offerers/api.py

```diff
@@ -1893,8 +1894,27 @@ def invite_members(offerer: models.Offerer, emails: list[str], current_user: use
     )  # TODO à modifier envoyer email seulement aux nouveaux invités
 
 
-def get_offerer_members(offerer: models.Offerer) -> list[models.UserOfferer]:
-    return models.UserOfferer.query.filter_by(offererId=offerer.id).join(users_models.User).all()
+def get_offerer_members(offerer: models.Offerer) -> list[typing.Tuple[str, OffererMemberStatus]]:
+    users_offerers = (
+        models.UserOfferer.query.filter(
+            models.UserOfferer.offererId == offerer.id, models.UserOfferer.validationStatus != ValidationStatus.REJECTED
+        )
+        .join(users_models.User)
```

Comment:
Comme tu n'as besoin que de l'email, pour réduire le volume extrait de la base, ne rapatrie que la colonne `email` :
```suggestion
        .options(sa.orm.joinedload(models.UserOfferer.user).load_only(users_model.User.email))
```